src/Symfony/Component/Notifier/Bridge/LineNotify/Tests/LineNotifyTransportTest.php

```diff
@@ -48,16 +48,6 @@ public function unsupportedMessagesProvider(): iterable
         yield [$this->createMock(MessageInterface::class)];
     }
 
-    public function testSendChatMessageWithMoreThan2000CharsThrowsLogicException()
-    {
-        $transport = $this->createTransport();
-
-        $this->expectException(LengthException::class);
-        $this->expectExceptionMessage('The subject length of a Line message must not exceed 1000 characters.');
-
-        $transport->send(new ChatMessage(str_repeat('囍', 1001)));
-    }
```

Comment:
This Test is obsolete, as we removed the check for the max subject length in the initial PR. Anyway, creating a ChatMessage with 1001 chars would never raise the limit 🤷 

Lets remove this test